### PR TITLE
Add script to bulk patch all devices in a search

### PR DESCRIPTION
**Pull request by woulamme**

### CODE REVIEW

File: bulk-patch.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          .then(pages => Array.prototype.concat.apply([], pages));
+      } else {
+        return result.data.items;
+      }
+    });
+};
+
+// Patch every device with settings from bulkSettings
+console.log('\nGetting devices...');
+getDevices(`https://api.ciscospark.com/v1/devices${devicesQuery}`).then(devices => {
```
Comment (kjeree) [inline bulk-patch.js]: Isn't `api.ciscospark.com` going away, in favor of `webexapis.com`? Ref. https://developer.webex.com/blog/introducing-the-new-webexapis-com

File: bulk-patch.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  console.log(`\nApplying ${bulkSettings.length} settings to ${devices.length} devices...`);
+  devices.forEach(device => {
+    const url = `https://api.ciscospark.com/v1/deviceConfigurations/${device.id}`;
+    axios.patch(url, JSON.stringify(bulkSettings), patchConfig)
```
Comment (kjeree) [inline bulk-patch.js]: `JSON.stringify` could fail, and the `.catch` below will just swallow the exception. What about parsing it on the line above (inside a `try/catch`)?

File: bulk-patch.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+const axios = require("axios");
+const parseLinkHeader = require('parse-link-header');
```
Comment (kjeree) [inline bulk-patch.js]: You are mixing single and double quotes. `yarn prettier --write bulk-patch.js ` should fix all such issues.

**Review by kjeree — CHANGES_REQUESTED** [review]

**Review by kjeree — APPROVED** [review]

File: bulk-patch.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
+const axios = require("axios");
+const parseLinkHeader = require("parse-link-header");
+
+const ACCESS_TOKEN = "MTFhMTExZWUtMDE5YS00YWU1LWE3NGUtNmFjOTk2YTZhMzA0MjgyZTcyYzgtZWU0_PF84_1eb65fdf-9643-417f-9974-ad72cae0e10f";
```
Comment (phrida) [inline bulk-patch.js]: Should we hide the access token as we do in the other script?

**Pull request merged**